src/styles/more-light/theme.pcss

```diff
@@ -671,58 +672,65 @@
 .p-input-filled .p-cascadeselect:not(.p-disabled).p-focus {
   background-color: var(--surface-a);
 }
-
 .p-checkbox {
-  width: 20px;
-  height: 20px;
+  width: var(--check-box-size) !important;
+  height: var(--check-box-size) !important;
+  @apply flex items-center justify-center;
+}
+.p-checkbox input {
+  width: var(--check-box-size) !important;
+  height: var(--check-box-size) !important;
+}
+.p-checkbox-box {
+  display: none;
 }
 .p-checkbox .p-checkbox-box {
-  border: calc(var(--border-weight)*2) var(--border-style) var(--bluegray-100);
-  background: var(--surface-a);
-  width: 20px;
-  height: 20px;
-  color: var(--text-color);
-  border-radius: var(--border-radius);
+  border: calc(var(--border-weight)*2) var(--border-style) var(--bluegray-100) !important;
+  background: var(--surface-a) !important;
+  width: var(--check-box-size) !important;
+  height: var(--check-box-size) !important;
+  color: var(--text-color) !important;
+  border-radius: var(--border-radius) !important;
   transition: background-color 0.2s, color 0.2s, border-color 0.2s, box-shadow 0.2s;
 }
 .p-checkbox .p-checkbox-box .p-checkbox-icon {
   transition-duration: 0.2s;
-  color: var(--surface-a);
-  font-size: calc(var(--default-font-size)*0.875);
+  color: var(--surface-a) !important;
```

Comment:
why are we applying `!important` on some styles but not all?

Reply:
The design flaws made me do it, I had to override some styles so the checkboxes look similar as before